modules/apigee_m10n_teams/tests/response-templates/monetization-company.json.twig

```diff
@@ -0,0 +1,35 @@
+{#
+/**
+ * @file
+ *   Developer
+ *
+ * Usage:
+ *   @code {% include 'monetization-developer.json.twig' %} @endcode
+ *
+ * Variables:
+ * - developer: The user account for the developer.
```

Comment:
I wonder why this wouldn't be renamed to company or team.

Reply:
Good catch, renamed to company.